Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang
+WORKDIR /app
+COPY * ./
+RUN go build -o processorService
+EXPOSE 18080
+CMD ["./processorService"]
```

Comment:
suggest `CMD` -> `ENTRYPOINT` instead

Reply:
lost track of this PR. Updated. PTAL.

Any specific reason of using Entrypoint over Cmd?

Reply:
thanks,; no strong reason other than there aren't that many parameters users would modify while running the container